core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -75,22 +75,23 @@
   </ng-audio-recorder>
   <button class="btn oppia-audio-button"
           uib-tooltip="Re-take" tooltip-placement="bottom"
-          ng-if="!recorder.status.isRecording && audioBlob"
+          ng-if="!recorder.status.isRecording && audioBlob && !checkSavingProcess"
           ng-click="reRecord()">
     <i class="material-icons">&#xE028;</i>
   </button>
   <button class="btn oppia-audio-button"
           uib-tooltip="Save" tooltip-placement="bottom"
-          ng-if="!recorder.status.isRecording && audioBlob"
+          ng-if="!recorder.status.isRecording && audioBlob && !checkSavingProcess"
           ng-click="saveRecordedAudio()">
     <i class="material-icons">&#xE161;</i>
   </button>
   <button class="btn oppia-audio-button"
           uib-tooltip="Cancel" tooltip-placement="bottom"
-          ng-if="!recorder.status.isRecording && audioBlob"
+          ng-if="!recorder.status.isRecording && audioBlob && !checkSavingProcess"
           ng-click="cancelRecording()">
     <i class="material-icons">&#xE5C9;</i>
   </button>
+  <div style="text-align:center;padding-top:13px;" ng-if="checkSavingProcess">Saving...</div>
```

Comment:
In general, try not to add inline styles, use a class instead. (Also, in this case, I think you can find a relatable class for showing logs in the end of this file)

Reply:
solved this